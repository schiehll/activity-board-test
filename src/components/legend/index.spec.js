import React from 'react'
import renderer from 'react-test-renderer'
import Legend from 'components/legend'

describe('Legend', () => {
  it('should render as expected', () => {
    const tree = renderer.create(<Legend />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
