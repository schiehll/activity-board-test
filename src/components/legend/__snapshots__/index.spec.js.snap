// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Legend should render as expected 1`] = `
<ul
  className="sc-ifAKCX gUfnWU"
>
  <div
    className="sc-bZQynM gGPWev"
  >
    Less
  </div>
  <li
    className="sc-EHOje cYSFWv"
    color="#ebedf0"
  />
  <li
    className="sc-EHOje hGMkQd"
    color="#c6e48b"
  />
  <li
    className="sc-EHOje jUPQpM"
    color="#7bc96f"
  />
  <li
    className="sc-EHOje gdoxPC"
    color="#239a3b"
  />
  <li
    className="sc-EHOje bFqLjS"
    color="#196127"
  />
  <div
    className="sc-bZQynM gGPWev"
  >
    More
  </div>
</ul>
`;
